pkg/TIPA.py

```
@@ -158,593 +188,546 @@ class MessageFunc():
 
-                    # Break the loop as we've found the item information
-                    if self.debug_mode >= 1:
-                        print("Found! Breaking")
+        # Scrape the gamepedia item webpage for more item details
+        try:
+            URL2 = f"https://escapefromtarkov.gamepedia.com/{true_name}"
+            page2 = requests.get(URL2, timeout=10)
 
-                    found = True
+            if page2.status_code != 200:
+                raise Exception("Error Code on gamepedia request: ", page2.status_code)
```

Ruff (0.8.0)

Abstract `raise` to an inner function

(TRY301)